front/components/agent_builder/triggers/ScheduleEditionModal.tsx

```diff
@@ -0,0 +1,186 @@
+import {
```

Comment:
I think we want to align with the tools and knowledge patterns here for the form. Let's try to mimic what we did there.

Let's allow people to "save" no matter what and bubble up the validation error only on submit?